DS 1.2 (EDA)

lopharb opened the pull request.

Commit:
DS-1.2

Changes:
- visualized some correlations between data

Commit:
DS-1.2

Changes:
- fixed a typo in the filename
- added some backround for further analysis (like the information about different days of the week etc.)

Commit:
DS-1.2

Changes:
-some more graphs representing different information
-added another additional column revenue to learn something about the prices of the sold items

Commit:
DS-1.2

Changes:
- added some conclusions
- added a few more analytical plots
- preformed some time series analysis

Commit:
DS-1.2

Changes:

- added some more commentary
- cleaned the code a little bit

Commit:
DS-1.2

Changes:
- grouped values for ACF by month
- added respective commentary

foxxyanya commented on notebooks/eda.ipynb:
please check the lines length in notebooks in the future  (github doesn't show the whole line if it exceeds the cell width)

foxxyanya replied:
nit: columns-target relationship plots don't contain in their title information about the way you aggregate them (e.g. the revenue for the whole time each shop existed, average revenue per month, etc.). It's always better to note this, as not everyone will check you code

upd: ok, later in your plots it's noted

foxxyanya replied:
by the way, it seems that in the DQC notebook we forgot to check general duplicates in rows

Commit:
Pull the changes made in DS-1.1 fixes

Commit:
DS-1.2 better EDA

Changes:

- added new stats to the analysis
- looked up some new things for previous research
- added more plots and comments
- added list of possible features w/description and ways to extract them
